docs/manage/_get_support.rst

```diff
@@ -34,10 +34,15 @@ Troubleshoot Database Issues
 
 In case you receive the ``database disk image is malformed`` error in Terminal, try running these commands (note that **you must have the** ``sqlite3`` **command available on your system**).
 
-  .. code-block:: bash
+    .. code-block:: bash
+
+       cp db.sqlite3 db.sqlite3.backup
+       cp db.sqlite3-wal db.sqlite3.backup-wal
+       cp db.sqlite3-shm db.sqlite3.backup-shm
```

Comment:
maybe it's better to replace these three cp commands by these two
```
mkdir -p corrupted
cp db.sqlite3* corrupted
```

Reply:
It's not corrupted but suspected to be "malformed", this would be a good distinction to make clear.